src/components/datagrid/data_grid_inmemory_renderer.tsx

```diff
@@ -56,11 +56,7 @@ export interface OuiDataGridInMemoryRendererProps {
 function noop() {}
 
 function getElementText(element: HTMLElement) {
-  return 'innerText' in element
-    ? element.innerText
-    : // (this line left here to satisfy Prettier since a ts-ignore is used on the next line)
-      // @ts-ignore TypeScript thinks element.innerText always exists, however it doesn't in jest/jsdom environment
-      element.textContent || undefined;
+  return 'innerText' in element ? element.innerText : element.textContent || '';
```

Comment:
It looks like we're changing the return type to `string` from `string | undefined` - is this intentional, and does it have any effect in caller contexts?

Reply:
While i think it will impact the caller contexts, i think the previous behavior of returning undefined was wrong.